frontend/public/components/utils/resource-log.jsx

```diff
@@ -45,6 +50,24 @@ const LogControls = ({dropdown, onDownload, toggleFullscreen, isFullscreen, stat
       {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
     </div>
     <div className="co-toolbar__group co-toolbar__group--right">
+      {!_.isEmpty(podLogLinks) && _.map(_.sortBy(podLogLinks, 'metadata.name'), (link, i) => {
+        const namespace = resource.metadata.namespace;
+        const namespaceFilter = link.spec.namespaceFilter;
+        // if namespaceFilter, do not output link if namespace does not match namespaceFilter regex
+        if (namespaceFilter && namespace.search(namespaceFilter) === -1) {
```

Comment:
Does this need to be in a `try...catch` or an `_.attempt()`?
What if a provider gives you something invalid.

Reply:
Referencing this:
https://github.com/openshift/api/pull/380/files#diff-9754f97c9c90951cc7df5ac76b6eea86R36
We probably need to be very clear about the expected format of the regex in our example, as well as ensure we give a couple good examples.  This kind of thing can be annoying if you don't know if `^openshift-` is ok, or if `/^openshift-/g` is needed.

Reply:
@jhadvig fyi

Reply:
```suggestion
        if (namespaceFilter && namespace.search(new RegExp(namespaceFilter, 'g')) === -1) {
```